src/App.tsx

```diff
@@ -3,33 +3,54 @@ import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-type State = {};
+type State = {
+  query: string,
+};
 
 export class App extends React.Component<{}, State> {
-  state: State = {};
+  state: State = {
+    query: '',
+  };
+
+  queryChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    this.setState({
+      query: event.target.value,
+    });
+  };
+
+  movies = () => {
+    return moviesFromServer.filter(movie => {
+      return movie.title.toLowerCase().includes(this.state.query.toLowerCase())
+      || movie.description.toLowerCase().includes(this.state.query.toLowerCase());
+    });
```

Comment:
You can optimize this:
```suggestion
    return moviesFromServer.filter(({ title, description }) => {
      const lowerCaseQuery = this.state.query.toLowerCase();
      
      return title.toLowerCase().includes(lowerCaseQuery)
      	|| description.toLowerCase().includes(lowerCaseQuery);
    });
```